.github/workflows/build_wheels.yml

```diff
@@ -128,7 +128,7 @@ jobs:
         run: |
           RELEASE_VERSION=${{ steps.get_rc_version.outputs.RELEASE_VERSION }}
           RC_NUM=${{ steps.get_rc_version.outputs.RC_NUM }}
-          sed -i -e "s/${RELEASE_VERSION}/${RELEASE_VERSION}rc${RC_NUM}/g" apache_beam/version.py
+          sed -i -e "s/${RELEASE_VERSION}/${RELEASE_VERSION}_rc${RC_NUM}/g" apache_beam/version.py
```

Comment:
I think we should fix it the other way around, as the release version follows an established pattern: https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#standards-compliance-for-interoperability